### PR TITLE
Fix dropdown

### PR DESCRIPTION
**Pull request by mkz212**

### CODE REVIEW

**Comment by donavanbecker** [comment]
@mkz212 Can you resolve the conflict

**Comment by mkz212** [comment]
@donavanbecker @bwp91 I need your help with this. The function at the bottom works great. But. The point is to exclude from this function click on dropdown.

**Comment by bwp91** [comment]
There is a conflict here again, please can you fix this, and what is it you need help with?

File: ui/src/app/app.component.ts [hunk 1]
```diff
@@ -70,9 +70,8 @@ export class AppComponent {
     }
 
     // ensure the menu closes when we navigate
-    router.events.subscribe((event: any) => {
-      const target = event.target;
-      if (event instanceof NavigationEnd && !target.hasClass('dropdown-toggle')) {
+    router.events.subscribe((event) => {
```
Comment (bwp91) [inline ui/src/app/app.component.ts]:
```suggestion
    router.events.subscribe((event: any) => {
```

**Comment by mkz212** [comment]
```
// ensure the menu closes when we navigate
    router.events.subscribe((event) => {
      if (event instanceof NavigationEnd) {
        const dropdownMenu = window.document.querySelector('#navbarSupportedContent');
        const hamburgerMenuButton = window.document.querySelector('.navbar-toggler');
        if (dropdownMenu) {
          dropdownMenu.classList.remove('show');
          hamburgerMenuButton.classList.add('collapsed');
        }
      }
    });
```


this function works great. ( I added` hamburgerMenuButton.classList.add('collapsed');` because it is nessesery to animate hamburger icon to X icon) But... I want to exclude click on dropdown in menu (class="toggle-dropdown") because now instead of open dropdown it close menu.

**Comment by bwp91** [comment]
I think im confusing menus - can you take a video of the current issue?

**Comment by mkz212** [comment]
`const dropdownMenu` should be `const navbar` - I don't know why it was a dropdown because it refers to the main menu

IMG_4480

**Comment by bwp91** [comment]
In the code this is still a link

```
          <a class="nav-link dropdown-toggle" href="#" role="button">
            <i class="fa fa-cog pr-2"></i>Homebridge Settings<i class="fa fa-caret-down pl-2"></i>
          </a>
```

maybe it should be a button?

**Comment by mkz212** [comment]
Can you try, please? The goal is to make dropdown open/close instead of closing whole menu.

**Comment by mkz212** [comment]
@bwp91 Ok done! I changed to button. Can you merge?

**Comment by bwp91** [comment]
is it working now?

**Comment by mkz212** [comment]
> is it working now?

It should. Can you merge? And release next alpha?

**Pull request merged**